frontend/packages/dev-console/src/components/topology/TopologyPage.tsx

```diff
@@ -79,6 +88,12 @@ const TopologyPage: React.FC<Props> = ({
     exact: true,
   });
 
+  React.useEffect(() => setTopologyActiveView(showListView ? 'list' : 'topology'), [showListView]);
+
+  if (getTopologyActiveView() === 'list' && !showListView) {
+    return <Redirect to={`/topology/ns/${namespace}/list`} />;
```

Comment:
Need to take into consideration when there is no namespace:
```suggestion
    return <Redirect to={`/topology/${namespace ? `ns/${namespace}` : 'all-namespaces'}/list`} />;
```
OR
if no namespace, ignore the redirect altogether?

Reply:
Yes, we should consider the namespace. I will add it.